community/values/src/test/java/org/neo4j/values/storable/TimeZonesTest.java

```diff
@@ -73,14 +73,14 @@ void weSupportDeletedZoneIdEastSaskatchewan()
     /**
      * If this test fails, you have changed something in TZIDS. This is fine, as long as you only append lines to the end,
      * or add a mapping to a deleted timezone. You are not allowed to change the order of lines or remove a line.
-     * p>
+     * <p>
      * If your changes were legit, please change the expected byte[] below.
      */
     @Test
     void tzidsOrderMustNotChange() throws URISyntaxException, IOException
     {
         Path path = Paths.get( TimeZones.class.getResource( "/TZIDS" ).toURI() );
-        byte[] timeZonesInfo = Files.readAllBytes( path );
+        String timeZonesInfo = Files.readString( path ).replace( System.lineSeparator(), "\n" );
```

Comment:
Does Git change the line endings of `TZIDS` on checkout? Or why is this necessary? I'm suspicious about this one because reading the file as a string just like that is relying on platform encoding, and I'm not sure all encodings will let the sha256 of the file stay the same.

Reply:
Yes, the EOL of `TZIDS` are system-specific, at least on Windows. We can change that with `.gitattributes` if it is better.

The current `sha256` seems to be successful on *nix system, and with this PR the test run will succeed with exact `sha256` on windows, so I believe it is the same on all systems, since it only depends on the text + `\n`, and not on EOL.